<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartActiv</class>
 <widget class="QMainWindow" name="StartActiv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>asset/Ico96.png</normaloff>asset/Ico96.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QPushButton" name="Pause_B">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QLabel" name="TXT1">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="10">
     <widget class="QCustomPlot" name="Graph_Trais" native="true"/>
    </item>
    <item row="0" column="15">
     <widget class="Line" name="line_14">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="Line" name="line_15">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Start_B">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="B4">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="Line" name="line_16">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="Line" name="line_20">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="14">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="11">
     <widget class="Line" name="line_17">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="10">
     <widget class="QLabel" name="TXT2">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="0" column="14">
     <widget class="QPushButton" name="Menu_B">
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="Line" name="line_13">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="B6">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="6" column="11">
     <widget class="Line" name="line_18">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Connect_B">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="14">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="12">
     <widget class="QLabel" name="TXT3">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="14">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="9">
     <widget class="Line" name="line_19">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="12" colspan="3">
     <widget class="QWidget" name="Info" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QListWidget" name="TRAIS_IN">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>2000</width>
           <height>2000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Fixed</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
         <property name="batchSize">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="B5">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="6" column="13">
     <widget class="Line" name="line_21">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="8">
     <widget class="QWidget" name="EXTERN" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0">
        <widget class="QLabel" name="Line_Tex_2">
         <property name="text">
          <string>--</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="Line" name="line_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Line_Tex_3">
         <property name="text">
          <string>Point Max</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLCDNumber" name="NUM_3"/>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLCDNumber" name="NUM_1">
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Line_Tex_1">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLCDNumber" name="NUM_4">
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QLCDNumber" name="NUM_2">
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="Line" name="line_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="Line_Tex_4">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="Line" name="line_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="Line" name="line_12">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QPushButton" name="Save_B">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="5">
     <widget class="QCustomPlot" name="Graph_Vs" native="true"/>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QPushButton" name="B1">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9" colspan="2">
     <widget class="QPushButton" name="B2">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11" colspan="2">
     <widget class="QPushButton" name="B3">
      <property name="text">
       <string>###</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
